app/services/service_offering/add_to_portfolio_item.rb

```diff
@@ -47,6 +50,8 @@ def populate_missing_fields
 
     def create_icon(icon_id)
       service_offering_icon = TopologicalInventory.call { |topo| topo.show_service_offering_icon(icon_id) }
+      return if service_offering_icon.data.nil?
```

Comment:
Should we add test coverage around this?

Reply:
Added :+1: